install_local.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
```

Comment:
```suggestion
#!/usr/bin/env python3
```

Remove obsolete encoding declaration and replace with shebang, since this should be a self-executable script. Also, I suggest marking as executable with `chmod a+x install_local.py`, or `git update-index --chmod=+x install_local.py` if on Windows.